examples/text_labels_and_annotations/demo_text_rotation_mode.py

```diff
@@ -1,9 +1,15 @@
-"""
-=======================
+"""=======================
 Demo Text Rotation Mode
 =======================
 
+The axes' method `~.axes.Axes.text` takes an argument ``rotation_mode`` that
+controls the alignment and rotation of the text. If ``rotation_mode`` is
+``None`` or ``default`` the text will first be rotated and then aligned
+according to the horizontal and vertical alignments (``ha`` and ``va`` in the
+example). If ``rotation_mode`` is ``anchor`` the text is aligned before rotation.
```

Comment:
This line is too long. We have a limit of 80 characters per line enforced by flake.

Reply:
Fixed.